packages/e2e-test-utils-playwright/src/page/activate-plugin.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Activates an installed plugin.
+ *
+ * @this {import('./').PageUtils}
+ * @param {string} slug Plugin slug.
+ */
+export async function activatePlugin( slug ) {
```

Comment:
Thanks for working on this.

There are `activatePlugin` and `deactivatePlugin` utils for playwright already in the codebase (as part of `RequestUtils`), so these `PageUtils` ones shouldn't be needed:
https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-test-utils-playwright/src/request/plugins.ts